<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editRegisterWindow</class>
 <widget class="QMainWindow" name="editRegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>You Are Editing A Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type the title of the book you want to edit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>441</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>681</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="author">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="author_line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>681</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="startYear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Year</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startYear_line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>751</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="endYear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>End Year</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endYear_line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="editionYear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Edition Year</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="editionYearLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volumes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Volumes</string>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="volumes_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="genre">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genre_line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>451</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="magazine">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Magazine</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="magazine_line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>441</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="publisher">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Publisher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="publisher_line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>441</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="applyChanges">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>458</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <widget class="QLabel" name="bookNotFound">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Book not found!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
